### DISCUSSION TITLE
Strong lensing

### DISCUSSION THREAD

**Discussion by Lucia-Fonseca** [skypyproject/skypy#535] [Ideas]
To explore event rates within SkyPy coupled with the galaxy module, quasars and the galaxy-host connection.

@sibirrer please feel free to add any more information to this topic. Thanks!

**Comment by Lucia-Fonseca** [comment]
@sibirrer would you have any suggestion?

**Reply by sibirrer** [reply]
The idea (in the long run) would be to to have on one side a module that allows to compute event rates of transients (SNe, GW etc) coupled with the galaxy module (meaning we know in which galaxy and perhaps even the position within the galaxy). Such modules (per se nothing to do with strong lensing) can then be coupled with a lensing deflector population (either drawn from a velocity dispersion abundance function or from a halo mass function with a coupling to stellar mass). Integrating such modules could then lead to an entire population study and event rate predictions for strongly lensed transients.

Similar, if we have a quasar-host galaxy connection module, we can get a better picture of the quasar population and the host galaxies. Such couplings may also be relevant for training sets to search for lensed quasars (and also transients, static lenses etc)
